app/assets/stylesheets/revised/pages/info/about.scss

```diff
@@ -114,4 +90,14 @@ $hover-about-social-color: lighten($gray-light, 15%);
       }
     }
   }
+
+  .about-alumni-list {
+    list-style-position: outside;
```

Comment:
This fixes the positioning of alumni images, but that listing is still hidden via a `d-none` on the `ul`

Reply:
That was the intention 🙂 - hidden, but not removed. Thanks for fixing them even though they're invisible!